index.bs

```diff
@@ -3036,25 +3036,18 @@ To <dfn>find sources with common destinations and reporting origin</dfn> given a
     1. [=list/Append=] |source| to |matchingSources|.
 1. Return |matchingSources|.
 
-To <dfn>remove fake event reports for source</dfn> given an [=attribution source=] |source| and a [=moment=] |minTriggerTime|:
-1. If |source|'s [=attribution source/randomized response=] is null, return.
-1. [=set/Remove=] all elements from the [=event-level report cache=] whose [=event-level report/trigger time=] is greater than or equal to |minTriggerTime| and whose [=event-level report/source identifier=] is equal to |source|'s [=attribution source/source identifier=].
-
-To <dfn>remove sources with different max event states</dfn> given an [=attribution source=] |pendingSource|:
-1. If |pendingSource|'s [=attribution source/max event states=] is null, return.
-1. Let |matchingSources| be the result of running [=find sources with common destinations and reporting origin=] with |pendingSource|.
-1. [=list/iterate|For each=] |source| of |matchingSources|:
-    1. If |source|'s [=attribution source/max event states=] is null or equals |pendingSource|'s [=attribution source/max event states=], [=iteration/continue=].
-    1. [=Remove fake event reports for source=] with |source| and |pendingSource|'s [=attribution source/source time=].
-    1. [=set/Remove=] |source| from the [=attribution source cache=].
+To <dfn>check if source has legacy attribution scope fields</dfn> given an [=attribution source=] |source| and an [=attribution source=] |pendingSource|:
```

Comment:
I think `legacy` is potentially misleading as a term. Maybe `incompatible`?

Reply:
`stale` could also work

Reply:
I think incompatible covers all cases pretty well.